test/get-all-allowances.js

```diff
@@ -0,0 +1,52 @@
+const assert = require("chai").assert;
```

Comment:
Nit: name of file should probably be `get-all-rep-allowances`, to keep it consistent.